scripts/test_all_fuzzers.sh

```diff
@@ -46,7 +50,9 @@ do
     fi
 
     # Save disk space
-    cargo clean
+    if [ "$1" != "--no-clean" ]; then
+        cargo clean
```

Comment:
Maybe we'll want another way to save disk space (it'll be above 10gigs otherwise), a quick Google surfaced cargo-sweep:
https://github.com/holmgr/cargo-sweep